manifests/params.pp

```diff
@@ -67,7 +67,7 @@
   $email_from = "no-reply@${::domain}"
   $email_enabled = false
 
-  $lazy_enabled = false
+  $lazy_enabled = true
```

Comment:
If Pulp doesn;t enable this by default, neither should the module. This should be up to users of the module such as puppet-katello to enable.